src/components/LandingPage/LandingPage.scss

```diff
@@ -115,12 +128,16 @@ $btn-hover: #4a99c0;
         left: 53%;
         transform: translate(-50%, -193%);
     }
+
+    .company-name{
+        font-size: 31px !important;
+    }
 }
 
 @media only screen and (max-width:480px) {
 
     .company-name {
-        font-size: 22px;
+        font-size: 22px !important;
```

Comment:
please explain change to '!important'?

Reply:
I changed it to the code to " !important  " because it was not picking up the the font-size when I change the browser-size on my computer. This corrected the responsive for your landing page along with the services page.